server/message.go

```diff
@@ -0,0 +1,57 @@
+package server
```

Comment:
Looks like this is being discussed as part of the https://github.com/coniks-sys/coniks-go/pull/29#discussion_r71884112 thread.